ClientScheduledExecutorServiceTest.schedule_whenShutdown

```
java.lang.AssertionError: Expected exception: java.util.concurrent.RejectedExecutionException
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:32)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:88)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:82)
	at java.util.concurrent.FutureTask.run(FutureTask.java:277)
	at java.lang.Thread.run(Thread.java:785)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-IbmJDK1.8/com.hazelcast$hazelcast-client/439/testReport/junit/com.hazelcast.client.scheduledexecutor/ClientScheduledExecutorServiceTest/schedule_whenShutdown/

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-ZuluJDK7/com.hazelcast$hazelcast-client/424/testReport/junit/com.hazelcast.client.scheduledexecutor/ClientScheduledExecutorServiceTest/schedule_whenShutdown/

--- a/hazelcast-client/src/main/java/com/hazelcast/client/proxy/ClientScheduledExecutorProxy.java
+++ b/hazelcast-client/src/main/java/com/hazelcast/client/proxy/ClientScheduledExecutorProxy.java
@@ -68,6 +68,7 @@ public class ClientScheduledExecutorProxy
         implements IScheduledExecutorService {
 
     private static final int GET_ALL_SCHEDULED_TIMEOUT = 10;
+    private static final int SHUTDOWN_TIMEOUT = 10;
 
     private static final FutureUtil.ExceptionHandler WHILE_SHUTDOWN_EXCEPTION_HANDLER =
             logAllExceptions("Exception while ScheduledExecutor Service shutdown", Level.FINEST);
@@ -301,7 +302,7 @@ public class ClientScheduledExecutorProxy
             calls.add(doSubmitOnAddress(request, SUBMIT_DECODER, member.getAddress()));
         }
 
-        waitWithDeadline(calls, 1, TimeUnit.SECONDS, WHILE_SHUTDOWN_EXCEPTION_HANDLER);
+        waitWithDeadline(calls, SHUTDOWN_TIMEOUT, TimeUnit.SECONDS, WHILE_SHUTDOWN_EXCEPTION_HANDLER);
     }
 
     private <T> ScheduledRunnableAdapter<T> createScheduledRunnableAdapter(Runnable command) {

